docker_secrets.json

```diff
@@ -5,5 +5,9 @@
   "DB_USER": "",
   "DB_HOST": "",
   "DB_PORT": "",
-  "DB_PASSWORD": ""
+  "DB_PASSWORD": "",
+  "ALLOWED_HOSTS": [],
+  "SITE_ID": "",
```

Comment:
Usually SITE_ID is an int, not sure if we should change it from empty string.

Reply:
Yeah...I'll have it changed.

Reply:
I think that would be ok. What do you think @somexpert ?

Reply:
I think it's fine. We're certainly not advocating that people deploy this into a production setting using that Docker setup.